xml/gl.xml

```diff
@@ -6890,6 +6890,15 @@ typedef unsigned int GLhandleARB;
         <enum value="0x96AC" name="GL_SHADING_RATE_4X2_PIXELS_QCOM" group="ShadingRateQCOM"/>
         <enum value="0x96AD" name="GL_SHADING_RATE_2X4_PIXELS_QCOM" group="ShadingRateQCOM"/>
         <enum value="0x96AE" name="GL_SHADING_RATE_4X4_PIXELS_QCOM" group="ShadingRateQCOM"/>
+        <enum value="0x96A6" name="GL_SHADING_RATE_1X1_PIXELS_EXT" alias="GL_SHADING_RATE_1X1_PIXELS_QCOM"/>
+        <enum value="0x96A7" name="GL_SHADING_RATE_1X2_PIXELS_EXT" alias="GL_SHADING_RATE_1X2_PIXELS_QCOM"/>
+        <enum value="0x96A8" name="GL_SHADING_RATE_2X1_PIXELS_EXT" alias="GL_SHADING_RATE_2X1_PIXELS_QCOM"/>
+        <enum value="0x96A9" name="GL_SHADING_RATE_2X2_PIXELS_EXT" alias="GL_SHADING_RATE_2X2_PIXELS_QCOM"/>
+        <enum value="0x96AA" name="GL_SHADING_RATE_1X4_PIXELS_EXT" alias="GL_SHADING_RATE_1X4_PIXELS_QCOM"/>
+        <enum value="0x96AB" name="GL_SHADING_RATE_4X1_PIXELS_EXT" alias="GL_SHADING_RATE_4X1_PIXELS_QCOM"/>
+        <enum value="0x96AC" name="GL_SHADING_RATE_4X2_PIXELS_EXT" alias="GL_SHADING_RATE_4X2_PIXELS_QCOM"/>
+        <enum value="0x96AD" name="GL_SHADING_RATE_2X4_PIXELS_EXT" alias="GL_SHADING_RATE_2X4_PIXELS_QCOM"/>
+        <enum value="0x96AE" name="GL_SHADING_RATE_4X4_PIXELS_EXT" alias="GL_SHADING_RATE_4X4_PIXELS_QCOM"/>
```

Comment:
```suggestion
        <enum value="0x96A6" name="GL_SHADING_RATE_1X1_PIXELS_EXT" alias="GL_SHADING_RATE_1X1_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96A7" name="GL_SHADING_RATE_1X2_PIXELS_EXT" alias="GL_SHADING_RATE_1X2_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96A8" name="GL_SHADING_RATE_2X1_PIXELS_EXT" alias="GL_SHADING_RATE_2X1_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96A9" name="GL_SHADING_RATE_2X2_PIXELS_EXT" alias="GL_SHADING_RATE_2X2_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96AA" name="GL_SHADING_RATE_1X4_PIXELS_EXT" alias="GL_SHADING_RATE_1X4_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96AB" name="GL_SHADING_RATE_4X1_PIXELS_EXT" alias="GL_SHADING_RATE_4X1_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96AC" name="GL_SHADING_RATE_4X2_PIXELS_EXT" alias="GL_SHADING_RATE_4X2_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96AD" name="GL_SHADING_RATE_2X4_PIXELS_EXT" alias="GL_SHADING_RATE_2X4_PIXELS_QCOM" group="ShadingRate"/>
        <enum value="0x96AE" name="GL_SHADING_RATE_4X4_PIXELS_EXT" alias="GL_SHADING_RATE_4X4_PIXELS_QCOM" group="ShadingRate"/>
```

Note: the "promotion" of enums from extension-specific groups to non-extension enums is undefined, likewise there's nothing defined against having EXT enums in a QCOM group and vice versa. To prevent this down the road, I suggest that we introduce a generic group in lieu of us formalizing how we go about this.

Reply:
Yeah, I wasn't quite sure how to do this aliasing. I tried a couple of different ways, but landed on this as it generates the right thing and keeps the definition of the enum values where they are reserved.